native/compress/build.gradle

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Hyperledger Besu Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+    id 'com.jfrog.artifactory' version '4.20.0'
+}
+
+repositories {
+    mavenCentral()
+}
+
+test {
+    useJUnitPlatform()
+}
+
+tasks.register('callBuildSh', Exec) {
+    workingDir buildscript.sourceFile.parentFile
+    commandLine 'bash', '../build.sh'
+}
+
+compileJava{
+    dependsOn callBuildSh
+}
+
+dependencies {
+    implementation 'net.java.dev.jna:jna:5.12.1'
+    testImplementation 'net.java.dev.jna:jna:5.12.1'
+    implementation 'org.apache.tuweni:tuweni-bytes:2.2.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.10.2'
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.10.2'
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+    testImplementation 'org.mockito:mockito-core:4.4.0'
```

Comment:
versions should go into `dependency-management.gradle`

Reply:
done